lib/2wp-utils.js

```diff
@@ -309,6 +309,24 @@ const get2wpBalances = async (rskTxHelper, btcTxHelper) => {
     
 };
 
+/**
+ * Creates a rejected_pegin event with the specified parameters.
+ * @param {Object} partialExpectedEvent an object with some rejected_pegin event default values.
+ * @param {string} btcPeginTxHash the pegin btc tx hash expected to be in the event.
+ * @param {string} rejectionReason the pegin rejection reason.
+ * @returns {BridgeEvent}
+ */
+const createExpectedRejectedPeginEvent = (partialExpectedEvent, btcPeginTxHash, rejectionReason) => {
```

Comment:
Receiving the `partialExpectedEvent` looks confusing

Why not use directly `PEGIN_EVENTS.REJECTED_PEGIN` inside the method? That will always be the param here, that's the event you are creating

Reply:
Updated.